geotools/src/main/scala/geotrellis/geotools/GridCoverage2DConverters.scala

```diff
@@ -190,6 +196,15 @@ object GridCoverage2DConverters {
     new Envelope2D(null, xmin, ymin, (xmax - xmin), (ymax - ymin))
   }
 
+  def getGeotoolsCRS(crs: CRS): CoordinateReferenceSystem = {
+    crs.epsgCode match {
+      case Some(code) =>
+        authorityFactory.createCoordinateReferenceSystem(s"EPSG:${code}")
```

Comment:
Why the `GeoToolsCRS` => `GeoToolsCRS.getAuthorityFactory(true).createCoordinateReferenceSystem(s"EPSG:${code}")` change?